.eslintrc.cjs

```diff
@@ -22,7 +22,8 @@ module.exports = {
 
 		// ESLint does not understand `import ... with { ... }`.
 		// See: https://github.com/eslint/eslint/discussions/15305.
-		'packages/ckeditor5-dev-ci/lib/data/index.js'
+		'packages/ckeditor5-dev-ci/lib/data/index.js',
+		'packages/ckeditor5-dev-build-tools/tests/build/fixtures/src/input.ts'
```

Comment:
Does Eslint throw an error with the new syntax? If so, we will have the error in other places in the future.

Reply:
ESLint expects `;` instead of `with`.